homepage/pelicanconf.py

```diff
@@ -75,6 +75,8 @@
 LINKEDIN_URL = 'https://www.linkedin.com/company/hioffen'
 TWITTER_URL = 'https://twitter.com/hioffen'
 RELEASE_DIRECT_URL = 'https://get.offen.dev'
+DOCKER_DIRECT_URL = 'https://hub.docker.com/r/offen/offen'
 HEROKU_DIRECT_URL = 'https://heroku.com/deploy?template=https://github.com/offen/heroku/tree/master'
+OFFEN_AUDITORIUM_URL = 'https://analytics.offen.dev/'
```

Comment:
Should this rather be `https://analytics.offen.dev/auditorium`?

Reply:
Yes bud what if user is not opt in?